tests/Integration/Action/SetupListener.php

```diff
@@ -58,7 +58,7 @@ public function startTestSuite(TestSuite $suite): void
 
         $drupalInstallationsCount = 0;
 
-        $process = new Process(\sprintf('%s %s/core/scripts/drupal install minimal', $phpBinary, $drupalDirectoryPath));
+        $process = new Process(\sprintf('%s %s/core/scripts/drupal install minimal', $phpBinary, $drupalDirectoryPath), null, null, null, null);
```

Comment:
The default timeout of 60 second is too short depending on the machine. For instance on my machine tests always fail because of this. It is the same for the docker container.